setup_env_convfwd_spr_mb56_thr56.sh

```diff
@@ -0,0 +1,126 @@
+# Usage:
+# <to be added>
+
```

Comment:
Name of the new file should probably be setup_env_convfwd_spr.sh since `mb` and `thr` are covered by if-statements inside.

Reply:
One of the yet not disccused aspect is the compiler/env dependency: whether we want to reflect dependency on the compiler (icc/gcc/clang), OpenMP RT and 

If we care a lot about reproducibility we should probably dump the parlooper and libxsmm code revision and environment like KMP_AFFINITY,GOMP_CPU_AFFINITY and who knows what else (page size? cache/memory BW specs?)